src/middlewared/middlewared/plugins/config.py

```diff
@@ -28,6 +32,7 @@ class ConfigService(Service):
         'configsave',
         Bool('secretseed', default=False),
         Bool('pool_keys', default=False),
+        Bool('authorized_keys', default=False),
```

Comment:
Maybe root_authorized_keys to make it clearer?

Also need the docstring updated.